README.md

```diff
@@ -49,6 +49,28 @@ Add `pod 'Yams'` to your `Podfile`.
 
 Add `github "jpsim/Yams"` to your `Cartfile`.
 
+### Bazel
+
+In your WORKSPACE file
+
+```WORKSPACE
+YAMS_GIT_SHA = "SOME_SHA"
+http_archive(
+    name = "yams",
+    urls = [
+        "https://github.com/jpsim/Yams/archive/%s.zip" % GTM_GIT_SHA
+    ],
+    strip_prefix = "yams-%s" % YAMS_GIT_SHA
+)
+
+load(
+    "@yams//bazel:repositories.bzl",
+    "yams_rules_dependencies",
+)
+
+yams_rules_dependencies()
```

Comment:
These aren't rules, but target definitions—I don't think this is needed. The workspace that loads `Yams` should already loads all the necessary dependencies declared in the `repositories.bzl` file.

```suggestion
```